packages/xdl/src/detach/IosShellApp.js

```diff
@@ -252,9 +263,16 @@ async function configureAndCopyArchiveAsync(args) {
       );
     } else if (type === 'archive' || type === 'client') {
       const workspaceName = type === 'archive' ? EXPOKIT_APP : EXPONENT_APP;
+      if (context.build.ios.bundleExecutable !== workspaceName) {
+        await spawnAsync('/bin/mv', [workspaceName, context.build.ios.bundleExecutable], {
```

Comment:
why not use fs package here?

Reply:
I don't know, I prefer not to know. Check out few lines below :)